src/nextjs-front/components/Chat.tsx

```diff
@@ -27,7 +29,27 @@ const Chat: React.FC<ChatProps> = ({ viewStack, onClose }) => {
 	}
 
 	return (
-		<div
+
+	<Draggable
+		position={{x: lastX, y: lastY}}
+		onStop={(e, data) => {
+			if (data.y > 0)
+		  		setLastY(0)
+			else if (-data.y > window.innerHeight - 670)
+		  		setLastY(-window.innerHeight + 670)
+			else
+		  		setLastY(data.y);
+
+			if (data.x > 0)
+			  	setLastX(0)
+			else if (-data.x > window.innerWidth - 550)
+		  		setLastX(-window.innerWidth + 550)
+			else
+		  		setLastX(data.x);
+			}}
+		disabled={useMediaQuery({ query: "(max-width: 800px)"})}
```

Comment:
I have this error  (sorry, approved by mistake and can't revoke approval):

![Screenshot at 2022-04-03 16-28-35](https://user-images.githubusercontent.com/41596925/161432875-6e118fca-ebbf-450d-b65d-08aab0c3a01c.png)